test/test.js

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+'use strict';
```

Comment:
We only need it in non-imported code (tests, example). When users import `puppeteer` those are run as modules and have an implicit 'use strict'`.